src/analysis/timeline/distributionToCred.js

```diff
@@ -49,14 +56,17 @@ export function distributionToCred(
     }
     cred[i] = new Array(intervals.length);
   }
-  if (scoringNodeIndices.length === 0) {
-    throw new Error("no nodes matched scoringNodePrefix");
-  }
 
   return ds.map(({interval, distribution, intervalWeight}) => {
     const intervalTotalScore = sum(
       scoringNodeIndices.map((x) => distribution[x])
     );
+
+    if (intervalTotalScore === 0) {
+      const cred = distribution.map((x) => x * 0);
```

Comment:
`((x) => x * 0)` is the same as `(() => 0)` for finite values

however, rather than special casing the return, I think it would be cleaner to say:

`const intervalNormalizer = intervalTotalScore == 0 ? 0 : intervalWeight / intervalTotalScore`

Reply:
Updated